packages/instrumenter/test/unit/mutators/conditional-expression-mutator.spec.ts

```diff
@@ -70,6 +70,50 @@ describe(sut.name, () => {
     );
   });
 
+  it('should not mutate (a || b) condition to (a || true)', () => {
```

Comment:
This shouldn't be in this PR, probably need to merge master into this branch.

Reply:
Right, I'm removing this.